### PR TITLE
feat(route): 知识星球

### LINKED ISSUE #5751: 知识星球
### 网站地址
https://wx.zsxq.com/
### 网站描述
知识星球，一个社群网站
### 需要生成什么内容？
指定社群的最新帖子

### PR DESCRIPTION
**Pull request by KarasuShin**
## Involved Issue / 该 PR 相关 Issue

Close #5751 

## Example for the Proposed Route(s) / 路由地址示例

```routes
/zsxq/group/88855458825252
/zsxq/user/2414218251
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
demo see [private rsshub](http://18.181.146.236:1200/zsxq/group/88855458825252)

### CODE REVIEW

File: lib/routes/zsxq/utils.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            default:
+        }
+        return {
+            title: topic.title ?? title,
+            description,
+            pubDate: parseDate(topic.create_time),
+        };
+    });
+}
+('');
```
Comment (TonyRL) [inline lib/routes/zsxq/utils.ts:71]: https://deepscan.io/dashboard/#view=project&tid=6244&pid=8135&bid=92448&subview=pull-request&prid=1895707

File: lib/routes/zsxq/group.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    return {
+        title: `知识星球 - ${group.name}`,
+        description: group.description,
+        image: group.background_url,
+        link: `https://wx.zsxq.com/dweb2/index/group/${groupId}`,
+        item: generateTopicDataItem(topics),
+    };
+}
+('');
```
Comment (TonyRL) [inline lib/routes/zsxq/group.ts:64]: https://deepscan.io/dashboard/#view=project&tid=6244&pid=8135&bid=92448&subview=pull-request&prid=1895707

File: lib/routes/zsxq/utils.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    throw new Error('something wrong');
+}
+
+function parseTopicContent(text: string = '', images: TopicImage[] = []) {
+    let result = text.replaceAll('\n', '<br>');
+    result = result.replaceAll(/<e type="web" href="(.*?)" title="(.*?)" \/>/g, (_, p1, p2) => `<a href=${decodeURIComponent(p1)}>${decodeURIComponent(p2)}</a>`);
+    result = result.replaceAll(/<e type="hashtag".*?title="(.*?)" \/>/g, (_, p1) => {
+        const title = decodeURIComponent(p1);
+        return `<span>${title}</a>`;
```
Comment (TonyRL) [inline lib/routes/zsxq/utils.ts:31]:
```suggestion
        return `<span>${title}</span>`;
```

**Comment by KarasuShin** [comment]
@TonyRL fixed. thanks for your review

**Pull request merged**